app/assets/stylesheets/master.css.scss

```diff
@@ -1,3 +1,12 @@
+.background-image {
+  background: url('cracked_paper.jpg') no-repeat center center fixed;
+  -webkit-background-size: cover;
```

Comment:
Properties should be ordered background, background-attachment, -moz-background-size, -o-background-size, -webkit-background-size, background-size<br>Avoid vendor prefixes.